build.sbt

```diff
@@ -51,8 +59,7 @@ SbtScalariform.scalariformSettings
 
 ScalariformKeys.preferences := ScalariformKeys.preferences.value
   .setPreference(AlignSingleLineCaseStatements, true)
-  .setPreference(DoubleIndentClassDeclaration, true)
-  .setPreference(PreserveDanglingCloseParenthesis, true)
```

Comment:
This leaves to reformatting a lot of source code, and not to the better. Is there any reason this is necessary?

Reply:
this seems to work: .setPreference(DanglingCloseParenthesis, scalariform.formatter.preferences.Preserve)